README.md

```diff
@@ -158,6 +161,32 @@ gql: {
 ```
 where `Page` and `UploadFile` are your type names for the **Content Types** you're referring by navigation items relations. 
 
+## i18n Internationalization
+
+### Settings
+
+This feature is **opt-in**.
+
+In order to use this functionality setting **default locale** is required. (See: Settings -> Internationalization)
+
+Once feature is enabled a restart is required. On server startup missing navigations for other locales will be created. From then you can manage navigation's localizations just like before.
+
+If you want go back to _pre-i18n_ way you can disable it in settings. Already created navigations will not be removed unless you make a choice for plugin to do so(this will require a restart).
+
+### Rendering
+
+Shape of the rendered navigation will not change. Querying stays almost the same. To query for specific locale version just add `locale` query param. For example:
+
+```https://yourdomain.cool/api/navigation/render/1?locale=fr```
+
+or
+
+```https://yourdomain.cool/api/navigation/render/main-navigation?locale=fr```
```

Comment:
as you're adding an example here it's worth to put also the `locale` property in the proper `render` chapter + GQL also (if supported already)